File: Selenium/Activities/Activity2.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity2 {

	public static void main(String[] args) {
		WebDriver driver=new ChromeDriver();
		
		driver.get("https://v1.training-support.net/selenium/login-form");
		System.out.println(driver.getTitle());
		
		driver.findElement(By.id("username")).sendKeys("admin");
		driver.findElement(By.id("password")).sendKeys("password");

		driver.findElement(By.xpath("//*[@id=\"dynamic-attributes-form\"]/div/button")).click();
		String message= driver.findElement(By.id("action-confirmation")).getText();
		System.out.println("LogIn message: " + message );
		
		driver.quit();
	}

}

File: Selenium/Activities/activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/login-form")
    print(driver.title)

    driver.find_element(By.ID, 'username').send_keys('admin')
    driver.find_element(By.ID, 'password').send_keys('password')
    driver.find_element(By.XPATH, '//*[@id=\"dynamic-attributes-form\"]/div/button').click()
    message= driver.find_element(By.ID, 'action-confirmation')
    print(message.text)

    driver.close()